tests/vm_test.go

```diff
@@ -1969,12 +1969,25 @@ status:
 
 			By("Blocking virt-handler from reconciling the VMI")
 			libpod.AddKubernetesApiBlackhole(getHandlerNodePod(virtClient, nodeName), componentName)
-			Eventually(getHandlerNodePod(virtClient, nodeName).Items[0], 120*time.Second, time.Second, HaveConditionFalse(k8sv1.PodReady))
+			Eventually(func() k8sv1.Pod {
+				return getHandlerNodePod(virtClient, nodeName).Items[0]
+			}, 120*time.Second, time.Second).Should(HaveConditionFalse(k8sv1.PodReady))
 
-			DeferCleanup(func() {
+			defer func() {
 				libpod.DeleteKubernetesApiBlackhole(getHandlerNodePod(virtClient, nodeName), componentName)
-				Eventually(getHandlerNodePod(virtClient, nodeName).Items[0], 120*time.Second, time.Second, HaveConditionTrue(k8sv1.PodReady))
-			})
+				Eventually(func() k8sv1.Pod {
+					return getHandlerNodePod(virtClient, nodeName).Items[0]
+				}, 120*time.Second, time.Second).Should(HaveConditionTrue(k8sv1.PodReady))
+
+				// FIXME: this is just a test to see if the flakiness is reduced
+				migrationBandwidth := resource.MustParse("1Mi")
+				kv := util.GetCurrentKv(virtClient)
+				kv.Spec.Configuration.MigrationConfiguration = &v1.MigrationConfiguration{
+					BandwidthPerMigration: &migrationBandwidth,
+				}
+				kv = testsuite.UpdateKubeVirtConfigValue(kv.Spec.Configuration)
+				tests.WaitForConfigToBePropagatedToComponent("kubevirt.io=virt-handler", kv.ResourceVersion, tests.ExpectResourceVersionToBeLessEqualThanConfigVersion, 120*time.Second)
```

Comment:
@jcanocan Please remove this.

Reply:
As far as I was able to test, this has been the key piece to stabilize the test. I've taken it from this other test: https://github.com/kubevirt/kubevirt/blob/7cfc6256f85e2853c9571f60e0b1087775cb820b/tests/virt_control_plane_test.go#L271-L279
Dropped the `FIXME` comment.

Reply:
I will give the flakes line a couple of extra runs to be sure.